Gallery/Panels/NCL_panel_4.py

```diff
@@ -68,8 +72,13 @@ def plot_labelled_filled_contours(data, ax=None):
         tick labels.
     """
 
-    # Import an NCL colormap, truncating it by using geocat.viz.util convenience function
-    newcmp = gv.truncate_colormap(cmaps.gui_default, minval=0.03, maxval=0.9)
+    # Import an NCL colormap, truncate it, and save the colormap
+
+    # Import the colormap
+    cmap = cmaps.gui_default
+    # Create a linear segmented colormap using the colormap
+    newcmp = mpl.colors.LinearSegmentedColormap.from_list(
+        name="newcmap", colors=cmap(np.linspace(0.03, 0.9, 100)))
```

Comment:
We might want to change the colormap here to reflect best practices as in other examples